### PR TITLE
Open user guides as external PDFs

### PR DESCRIPTION
**Pull request by netalondon**

### CODE REVIEW

File: web/src/shell/header.tsx [hunk 1]
```diff
@@ -63,19 +61,20 @@ const headerButtons: HeaderButton[] = [
...
+      context.appContext.settings.open();
     },
     icon: "settings",
     tooltip: "Settings",
   },
   headerButtonFromURL(URLs["about"], "info", "About"),
 ];
 
+const guideLinks: Record<string, string> = {
+  chip: "https://drive.google.com/file/d/15unXGgTfQySMr1V39xTCLTgGfCOr6iG9/view",
```
Comment (DavidSouther) [inline web/src/shell/header.tsx:74]: Neither of these include screenshots - are we sure we want these links? I think I'd suggest we put these in a versioned branch on Github to better track updates.

New bare branch: user-guide
Links: https://raw.githubusercontent.com/nand2tetris/web-ide/user-guide/chip.pdf, etc
Review process: Iterate on Drive, save as PDF, replace & commit in user-guide branch, upload.

Alternatively, use a URL shortener to better track downloads etc.

**Review by DavidSouther — APPROVED** [review]

**Comment by netalondon** [comment]
> Ooh neat - could you share a couple screenshots (the buttons, and the opened window)

Sure, the buttons haven't changed but here are the opened windows:

Screenshot from 2024-04-10 11-32-41
Screenshot from 2024-04-10 11-33-10

**Comment by netalondon** [comment]
> Ooh neat - could you share a couple screenshots (the buttons, and the opened window)

Sure, the buttons haven't changed but here are the opened windows:

Screenshot from 2024-04-10 11-32-41
Screenshot from 2024-04-10 11-33-10

File: web/src/shell/header.tsx (hunk 1 above)
Comment (netalondon) [reply web/src/shell/header.tsx]: Fixed

**Pull request merged**